plugins/dev-tools/README.md

```diff
@@ -13,6 +10,31 @@ npm install @blockly/dev-tools -D --save
 
 ## Usage
 
+### Playground
+The playground is a tremendously useful tool for debugging your Blockly project. As a preview, here is [one of the plugin playgrounds](https://google.github.io/blockly-samples/plugins/theme-modern/test/). The playground features are:
+- All the default blocks
+- All the language generators (JavaScript, Python, PHP, Lua, and Dart)
+- Switch between different Blockly options (eg: rtl, renderer, readOnly, zoom and scroll)
+- Switch between different toolboxes and themes
+- Import and export programs, or generate code using one of the built-in generators
+- Trigger programatic actions (eg: Show/hide, Clear, Undo/Redo, Scale)
```

Comment:
nit: programatic -> programmatic

Reply:
Has to be the most common developer typo..